src/main/kotlin/com/cognifide/gradle/aem/environment/docker/Stack.kt

```diff
@@ -0,0 +1,46 @@
+package com.cognifide.gradle.aem.environment.docker
+
+import org.buildobjects.process.ExternalProcessFailureException
+import org.buildobjects.process.ProcBuilder
```

Comment:
GAP already has many dependencies, if it is possible I am trying to avoid adding new ones.
What is more, I would like to even eliminate all possible: https://github.com/Cognifide/gradle-aem-plugin/issues/302